code/modules/clothing/hybrisa_clothing.dm

```diff
@@ -524,7 +607,7 @@
 
 // WY Pilot
 
-/obj/item/clothing/suit/storage/hybrisa/WY_Pilot
+/obj/item/clothing/suit/storage/hybrisa/wy_Pilot
```

Comment:
Do you still want it to be capital P?

Reply:
ah missed this one, but no I've changed all so far.